src/compiler/parser.ts

```diff
@@ -1518,6 +1518,132 @@ namespace ts {
             return false;
         }
 
+        const commonKeywords = [

```

Comment:
I just stumbled onto `textToKeywordObj` / `textToKeyword` in the scanner. @DanielRosenwasser / whoever-is-reviewing-this, what do you think about exposing one of those `@internal`ly and using them here?

Reply:
I went ahead and did this in 05e8d0ffc2c64cb55a8e8665d30084265f08da3d. 😄